common/lib/capa/capa/responsetypes.py

```diff
@@ -250,8 +250,16 @@ def render_html(self, renderer, response_msg=''):
           - renderer : procedure which produces HTML given an ElementTree
           - response_msg: a message displayed at the end of the Response
         """
-        # render ourself as a <span> + our content
-        tree = etree.Element('span')
+        RESPONSE_LABEL = _('Question')
+
+        # get responsetype index to make responsetype label
+        response_idx = self.xml.attrib['id'].split('_')[-1]
+
+        # wrap the content insode a section
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
+        tree.set('aria-label', RESPONSE_LABEL + ' ' + response_idx)
```

Comment:
We need some tests around this logic, more specifically about the value for aria-label for different scenarios/questions.